intermediate_notebooks/E2E/mortgage/mortgage_e2e.ipynb

```diff
@@ -0,0 +1,772 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Mortgage Workflow\n",
+    "\n",
+    "## The Dataset\n",
+    "The dataset used with this workflow is derived from [Fannie Mae’s Single-Family Loan Performance Data](http://www.fanniemae.com/portal/funding-the-market/data/loan-performance-data.html) with all rights reserved by Fannie Mae. This processed dataset is redistributed with permission and consent from Fannie Mae.\n",
+    "\n",
+    "To acquire this dataset, please visit [RAPIDS Datasets Homepage](https://docs.rapids.ai/datasets/mortgage-data)\n",
+    "\n",
+    "## Introduction\n",
+    "The Mortgage workflow is composed of three core phases:\n",
+    "\n",
+    "1. ETL - Extract, Transform, Load\n",
+    "2. Data Conversion\n",
+    "3. ML - Training\n",
+    "\n",
+    "### ETL\n",
+    "Data is \n",
+    "1. Read in from storage\n",
+    "2. Transformed to emphasize key features\n",
+    "3. Loaded into volatile memory for conversion\n",
+    "\n",
+    "### Data Conversion\n",
+    "Features are\n",
+    "1. Broken into (labels, data) pairs\n",
+    "2. Distributed across many workers\n",
+    "3. Converted into compressed sparse row (CSR) matrix format for XGBoost\n",
+    "\n",
+    "### Machine Learning\n",
+    "The CSR data is fed into a distributed training session with Dask-XGBoost"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Imports statements"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%env NCCL_P2P_DISABLE=1 # Necessary for NCCL < 2.4"
```

Comment:
I believe this is no longer needed, as XGBoost depends on a recent nccl conda package.